src/main/java/com/algorand/algosdk/v2/client/model/PendingTransactionResponse.java

```diff
@@ -0,0 +1,214 @@
+	 */
+	private long closingAmount;
+	private boolean closingAmountIsSet;
+	@JsonProperty("closing-amount")
+	public void setClosingAmount(long closingAmount){
+		this.closingAmount = closingAmount;
+		closingAmountIsSet = true;
+	}
+	@JsonProperty("closing-amount")
+	public Long getClosingAmount(){
+		return closingAmountIsSet ? closingAmount : null;
+	}
+	/**
+	 * Check if has a value for closingAmount 
+	 */	@JsonIgnore
+	public boolean hasClosingAmount(){
+		return closingAmountIsSet;
+	}
+
+	/**
+	 * The round where this transaction was confirmed, if present. 
+	 */
+	private long confirmedRound;
+	private boolean confirmedRoundIsSet;
+	@JsonProperty("confirmed-round")
+	public void setConfirmedRound(long confirmedRound){
+		this.confirmedRound = confirmedRound;
+		confirmedRoundIsSet = true;
+	}
+	@JsonProperty("confirmed-round")
+	public Long getConfirmedRound(){
+		return confirmedRoundIsSet ? confirmedRound : null;
+	}
+	/**
+	 * Check if has a value for confirmedRound 
+	 */	@JsonIgnore
+	public boolean hasConfirmedRound(){
+		return confirmedRoundIsSet;
+	}
+
+	/**
+	 * Indicates that the transaction was kicked out of this node's transaction pool 
+	 * (and specifies why that happened). An empty string indicates the transaction 
+	 * wasn't kicked out of this node's txpool due to an error. 
+	 */
+	private String poolError;
+	private boolean poolErrorIsSet;
+	@JsonProperty("pool-error")
+	public void setPoolError(String poolError){
+		this.poolError = poolError;
+		poolErrorIsSet = true;
+	}
+	@JsonProperty("pool-error")
+	public String getPoolError(){
+		return poolErrorIsSet ? poolError : null;
+	}
+	/**
+	 * Check if has a value for poolError 
+	 */	@JsonIgnore
+	public boolean hasPoolError(){
+		return poolErrorIsSet;
+	}
+
+	/**
+	 * Rewards in microalgos applied to the receiver account. 
+	 */
+	private long receiverRewards;
+	private boolean receiverRewardsIsSet;
+	@JsonProperty("receiver-rewards")
+	public void setReceiverRewards(long receiverRewards){
+		this.receiverRewards = receiverRewards;
+		receiverRewardsIsSet = true;
+	}
+	@JsonProperty("receiver-rewards")
+	public Long getReceiverRewards(){
+		return receiverRewardsIsSet ? receiverRewards : null;
+	}
+	/**
+	 * Check if has a value for receiverRewards 
+	 */	@JsonIgnore
+	public boolean hasReceiverRewards(){
+		return receiverRewardsIsSet;
+	}
+
+	/**
+	 * Rewards in microalgos applied to the sender account. 
+	 */
+	private long senderRewards;
+	private boolean senderRewardsIsSet;
+	@JsonProperty("sender-rewards")
+	public void setSenderRewards(long senderRewards){
+		this.senderRewards = senderRewards;
+		senderRewardsIsSet = true;
+	}
+	@JsonProperty("sender-rewards")
+	public Long getSenderRewards(){
+		return senderRewardsIsSet ? senderRewards : null;
+	}
+	/**
+	 * Check if has a value for senderRewards 
+	 */	@JsonIgnore
+	public boolean hasSenderRewards(){
+		return senderRewardsIsSet;
+	}
+
+	/**
+	 * The raw signed transaction. 
+	 */
+	private String txn;
```

Comment:
This should be our existing `SignedTransaction` object

Reply:
Can we have this information communicated via the spec file? 
Now it says: 
        "type": "object",

Reply:
fixed.